audit/projects/k8s-conform/services/enabled.txt

```diff
@@ -1,6 +1,4 @@
 NAME                              TITLE
-compute.googleapis.com            Compute Engine API
-oslogin.googleapis.com            Cloud OS Login API
```

Comment:
Expected, service disables are due to https://github.com/kubernetes/k8s.io/pull/1890#issuecomment-816326889